packages/typography/src/typography.scss

```diff
@@ -87,3 +62,28 @@
 .hig__typography__sub2 {
   @include typography-h5;
 }
+
+// COLORS
+.hig__typography--hig-white {
+  color: color('hig-white');
+}
+
+.hig__typography--hig-cool-gray-70 {
+  color: color('hig-cool-gray-70');
+}
+
+.hig__typography--hig-blue-50 {
+  color: color('hig-blue-50');
+}
+
+.hig__typography--hig-green-good {
+  color: color('hig-green-good');
+}
+
+.hig__typography--hig-yellow-warning {
+  color: color('hig-yellow-warning');
+}
+
+.hig__typography--hig-red-alert {
+  color: color('hig-red-alert');
+}
```

Comment:
Moved the color rules after the default rules, so colors would supersede the defaults.